CONTRIBUTING.md

```diff
@@ -1,10 +1,11 @@
-- [Contributing to OpenSearch-SDK-Java](#contributing-to-opensearch-sdk-java)
+- [Contributing to OpenSearch-SDK-Java](#contributing-to-opensearch-sdk)
```

Comment:
Should we keep the title below as `Contributing to OpenSearch-SDK-Java`?

Reply:
Probably "for java"....